tf_files/aws_squid_vm/outputs.tf

```diff
@@ -0,0 +1,5 @@
+output "squid_public_ip" {
+ value = "${module.squid_vm.squid_public_ip}"
+}
+
+
```

Comment:
Do you need the 'private_ip' as an output too, so it can be used in internal route tables?

Reply:
I don't need it for the route tables; but probably good to have it as a output for ssh access.I will add it.